CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ about why a change log is important.
 ### Removed
 - Drop support for Rails 5.2 (#164)
 - Drop support for Ruby 2.7 (#163)
+- Drop support for Rails 6.0 (#167)
```

Comment:
Please move this line to the top of the section so the order of the items is in reverse chronological order.